### PR TITLE
(build) Switch to using Web Targets NuGet Package

### PR DESCRIPTION
**Pull request by gep13**
- I didn't have the specific Web Targets on my VM
- Using this NuGet package means that it should build on any machine

### CODE REVIEW

**Comment by gep13** [comment]
@ferventcoder what are your thoughts on this addition?

File: src/SimpleChocolateyServer/Web.config [hunk 1]
```diff
@@ -73,7 +73,7 @@
     </handlers>
     <staticContent>
       <remove fileExtension=".ps1" />
-      <mimeMap fileExtension=".nupkg" mimeType="application/zip"/>
+      <mimeMap fileExtension=".nupkg" mimeType="application/zip" />
```
Comment (ferventcoder) [inline src/SimpleChocolateyServer/Web.config]: These are formatting changes - they go in a separate maint commit or are undone. The only time formatting changes should be with other changes is when it is unavoidable or part of the subject being changed.

Comment (gep13) [reply src/SimpleChocolateyServer/Web.config]: Argh, never spotted that. Will rebase and fix that.

File: src/SimpleChocolateyServer/SimpleChocolateyServer.csproj [hunk 2]
```diff
@@ -156,7 +157,6 @@
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(VSToolsPath)\WebApplications\Microsoft.WebApplication.targets" Condition="'$(VSToolsPath)' != ''" />
```
Comment (gep13) [inline src/SimpleChocolateyServer/SimpleChocolateyServer.csproj]: @ferventcoder the problem was here. When running build.bat, this resolved to a v11 folder, which I don't have on my VM as I only have VS 2017 installed.

Comment (ferventcoder) [reply src/SimpleChocolateyServer/SimpleChocolateyServer.csproj]: :+1:

**Comment by ferventcoder** [comment]
> Looks good, but why is this necessary when it uses _PublishedApplications targets? Do they make reference to this file?

Ignore this. I was thinking of something else when I left this comment.

**Comment by ferventcoder** [comment]
Once you have this rebased and ready, let me know and I'll merge it in.

**Comment by gep13** [comment]
@ferventcoder I believe I have made the necessary changes here.  Let me know if you spot anything else.

File: src/SimpleChocolateyServer/Web.config (hunk 1 above)
Comment (ferventcoder) [reply src/SimpleChocolateyServer/Web.config]: No worries here.

**Comment by ferventcoder** [comment]
Merged into master at https://github.com/chocolatey/simple-server/commit/639b28dfb8a7a68c36e3670780426904113ee16d

**Comment by ferventcoder** [comment]
No idea what the issue was, but I had to revert this just now. Once switched it does not properly create the _PublishedWebsites folder, causing running build.bat to break.

**Pull request merged**